examples/arduino/LVGL_Arduino/LVGL_Arduino.ino

```diff
@@ -67,6 +67,12 @@ void my_touchpad_read( lv_indev_t * indev, lv_indev_data_t * data )
      */
 }
 
+/*use Arduinos millis() as tick source*/
+uint32_t my_tick(void)
+{
+    return millis();
```

Comment:
LGTM
Should we add a force cast here for the return value of `millis()`?

Reply:
You mean `return (uint32_t) millis();`?
I tested it with Arduino IDE and PlatformIO. There is no difference. It works with and without the type cast. In both cases, no compiler warnings.